File: atCoder/123_arc_A.py
ans = 0
a, b, c = map(int, input().split())
if a>c:
	a,c = c,a
if (a+c) % 2:
	c += 1
	ans = 1
	
	
B = (a+c) / 2

if b <= B:
	ans += B - b
else:
	if ans:
		c -= 1
		a += 1
	ans += 2*b - (a + c)

print(int(ans))

File: atCoder/123_arc_A.cpp
#include <bits/stdc++.h>
using namespace std;

#define ll long long

int main() {
	ll a, b, c;
	cin >> a >> b >> c;
	if (a > c) swap(a, c);
	
	ll ans = 0;
	if (b >= c) {
		ans = (2 * (b-a) + a) - c;
	} else if (b <= a) {
		if ((c-a) % 2) {
			ans = 1;
			a++;
		}
		ans += (c-a)/2 + a - b;
	} else {
		
	}	
	cout << ans << endl; 
}
















/*
int main() {
	ll a, b, c, ans = 0;
	cin >> a >> b >> c;
	if (a > c) swap(a, c);
	
	if (a == c) {
		ans = abs(a-b);
	} else if ((a+c) % 2) {
		ans = min(abs((a+c-1) / 2 - b), abs((a+c+1) / 2 - b));
	} else {
		ans = abs((a+c) / 2 - b);
	}
	
	cout << ans << endl;
}
*/
